docs/usage.rst

```diff
@@ -439,24 +449,31 @@ To request specific nodes.
         outputs=['gender', 'confidence'],
     )
 
-Or a single node:
+To concatenate the outputs to a single array,
+do:
 
 .. jupyter-execute::
 
     onnx_model_7(
         signal,
         sampling_rate,
-        outputs='gender',
+        outputs=['gender', 'confidence'],
+        concat=True,
     )
 
 Create interface and process a file.
 
 .. jupyter-execute::
 
+    feature_names = onnx_model_7.outputs['gender'].labels + \
+        onnx_model_7.outputs['confidence'].labels
```

Comment:
In principle we can write this slightly more general, but the problem is it gets not better to read and requires the strange `[]` argument to `sum()`:

```python
    outputs = ['gender', 'confidence']
    feature_names = sum(
        onnx_model_7.outputs[o].labels for o in outputs,
        [],
    )
    interface = audinterface.Feature(
        feature_names=feature_names,
        process_func=onnx_model_7,
        process_func_args={
            'outputs': outputs,
            'concat': True,
        },
    )
    interface.process_file(file)
```

So maybe for the future we might think about a helper function:

```python
feature_names = onnx_model_7.labels(['gender', 'confidence'])
```

Reply:
Good idea.